package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "loopback-workspace",
-  "version": "4.5.0",
+  "version": "4.6.0",
```

Comment:
when we publish a release, does it modify the version automatically?

Reply:
right...I created a branch 4.x_latest for publishing the 4.x patches. When publishing there the version should change. Let me revert this.